src/apps/components/juttle-editor.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import AceEditor from 'juttle-react-ace';
```

Comment:
I like `react-ace`. I'd lean towards making further modifications to `juttle-react-ace` if we need any other functionality.